### PR TITLE
Add horse class

### PR DESCRIPTION
**Pull request by Pysyren**
Please check...

### CODE REVIEW

File: horse.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+function Horse(){
+
```
Comment (Pysyren) [inline horse.js]: Remove this line break

**Review by Pysyren — COMMENTED** [review]
Finish

File: horse.js (hunk 1 above)
Comment (KhanhVanCong) [reply horse.js]: you should remove this line

**Review by KhanhVanCong — COMMENTED** [review]
Nothing

**Pull request merged**